eth/api_debug_test.go

```diff
@@ -224,3 +230,30 @@ func TestStorageRangeAt(t *testing.T) {
 		}
 	}
 }
+
+func TestExecutionWitness(t *testing.T) {
+	t.Parallel()
+
+	// Create a database pre-initialize with a genesis block
+	db := rawdb.NewMemoryDatabase()
+	gspec := &core.Genesis{
+		Config: params.TestChainConfig,
+		Alloc:  types.GenesisAlloc{testAddr: {Balance: big.NewInt(1000000)}},
+	}
+	chain, _ := core.NewBlockChain(db, nil, gspec, nil, ethash.NewFaker(), vm.Config{}, nil)
+
+	blockNum := 10
+	_, bs, _ := core.GenerateChainWithGenesis(gspec, ethash.NewFaker(), blockNum, nil)
```

Comment:
Could you add some transactions to the test chain so that we have some more data in the witnesses other than the coinbase payment? 

e.g.
```
tx, err := types.SignTx(types.NewTransaction(0, common.Address{0x00}, new(big.Int), params.TxGas, block.BaseFee(), nil), signer, key)
			if err != nil {
				panic(err)
			}
			block.AddTx(tx)
```

Reply:
done